docs/ramp_fitting/source/main.rst

```diff
@@ -12,7 +12,7 @@ data cube in a second output data product.  A third, optional output product
 is also available and is produced only when the step parameter 'save_opt' is
 True (the default is False).  The output values will be in units of counts (DN)
```

Comment:
In what case(s) would the output values ever be in units of counts? I would think that even for pixels that only have 1 valid group, you then divide that value by the group time to get a slope (counts per second).